src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -49,3 +34,27 @@
     (utils/get-account-by-address accounts account-address)
     :balance
     (utils/get-balance-by-address balances account-address))))
+
+(defn- prepare-token
+  [{:keys [symbol marketValuesPerCurrency] :as item}]
```

Comment:
When tokens received in `:wallet/get-wallet-token-success` we can transform them using `camel-snake-kebab` package to avoid propagation of camel-case)

Will it be convenient for you to add this to your PR? As far as I can see, that won't lead to a lot of changes, only your code and content of `status-im2.subs.wallet.wallet-test`.

```clojure
(:require
    [camel-snake-kebab.core :as csk]
    [camel-snake-kebab.extras :as cske])
....
(rf/defn get-wallet-token-success
  {:events [:wallet/get-wallet-token-success]}
  [{:keys [db]} data]
  {:db (assoc db
              :wallet/tokens          (cske/transform-keys csk/->kebab-case-keyword data)   <---
              :wallet/tokens-loading? false)})
```

Reply:
@vkjr fixed !